aldryn_django/storage.py

```diff
@@ -26,16 +31,24 @@
 
 class S3MediaStorage(s3boto.S3BotoStorage):
     def __init__(self):
+        bucket_name = settings.AWS_MEDIA_STORAGE_BUCKET_NAME
+
+        if '.' in bucket_name:
+            calling_format = OrdinaryCallingFormat()
```

Comment:
I think bucket names with ``.`` are evil anyway, because they don't work with SSL by default on S3. But I could be wrong.